src/data_loader.py

```diff
@@ -0,0 +1,160 @@
+import os
+import PIL
+import torch
+import random
+import collections
+import numpy as np
+import pandas as pd
+
+
+
+class ImageSegmentaionDataset():
+    """Image Segmentation dataset"""
+    
+    def __init__(self, image_dict, loader_type, replacement=True, transform=None):
+        """
+        Args:
+        image_dict (dict): a dict of the following format:
+                        {"ships": {"imageId": "segmentation pixels"}， “w/ships”: ["imageId"]}
+        loader_type: (string)"classification" or "segmentation"
```

Comment:
Could you split up this dataset into two?
For example, ImageSegmentationDataset and ImageClassificationDataset. Right now the naming "ImageSegmentaionDataset" suggests only one type, so presence of "loader_type" is confusing.

It might make sense to have a base class with all common functionality and extend it to ImageSegmentation and ImageClassification ones, which will override type-specific methods only.

Reply:
Sorry for the confusion..... I like your suggestion, but in order to save time, I changed the name of the class to be AirbusDataset and still include the "loader_type".